backend/core/quivr_core/models.py

```diff
@@ -83,12 +83,6 @@ class ParsedRAGChunkResponse(BaseModel):
     last_chunk: bool = False
 
 
-class KnowledgeStatus(str, Enum):
```

Comment:
I think we should rollback this

Reply:
?? it is exactly the same as in release/quivr-core